assets/static/images/icon-late-view-ghost-with-waiver.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20.93 19.99"><defs><style>.cls-1,.cls-3,.cls-4{fill:#1c1e23;}.cls-2,.cls-5{fill:#fff;}.cls-2{stroke:#8b8d91;stroke-width:0.79px;}.cls-3,.cls-4{stroke:#1c1e23;stroke-width:0.53px;}.cls-3,.cls-5{fill-rule:evenodd;}</style></defs><circle class="cls-1" cx="17.4" cy="3.53" r="3"/><circle class="cls-2" cx="9.4" cy="10.59" r="9"/><path class="cls-3" d="M4.6,9.28c0-2.69,2.46-4.88,5.48-4.88s5.48,2.19,5.46,4.88v6.17a.88.88,0,0,1-.36.71,1,1,0,0,1-.8.13.2.2,0,0,1-.08,0l-1.56-.94-2.48,1.4a.32.32,0,0,1-.19.05.34.34,0,0,1-.19-.05l-2.47-1.4-1.57.94a.2.2,0,0,1-.08,0,.9.9,0,0,1-1.16-.84Zm10.14,6.25a.12.12,0,0,0,.06-.1V9.28c0-2.27-2.12-4.11-4.72-4.11S5.36,7,5.36,9.28v6.15a.14.14,0,0,0,.06.11l.08,0,1.71-1a.46.46,0,0,1,.2,0,.32.32,0,0,1,.19,0l2.48,1.4,2.48-1.41a.39.39,0,0,1,.39,0l1.71,1A.11.11,0,0,0,14.74,15.53Z" transform="translate(-0.6 -0.41)"/><circle class="cls-1" cx="8.66" cy="9.85" r="1.06"/><circle class="cls-1" cx="12.26" cy="9.85" r="1.06"/><circle class="cls-4" cx="17.4" cy="3.53" r="3.26"/><path class="cls-5" d="M18.48,1.34h-1a.24.24,0,0,0-.2.28l.2,3.09a.23.23,0,0,0,.2.23h.56a.23.23,0,0,0,.2-.23l.2-3.09a.24.24,0,0,0-.2-.28" transform="translate(-0.6 -0.41)"/><path class="cls-5" d="M18.32,5.34h-.64a.26.26,0,0,0-.25.26v.68a.25.25,0,0,0,.25.26h.64a.25.25,0,0,0,.25-.26V5.6a.26.26,0,0,0-.25-.26" transform="translate(-0.6 -0.41)"/></svg>
```

Comment:
Have you run this through `svgo`? I think we have a step in the webpack process that does so anyway but I like to run our SVGs through it by hand just to clean up the formatting and such to make it easier to read.